eth-types/src/lib.rs

```diff
@@ -398,6 +398,9 @@ pub struct ResultGethExecTrace {
 /// before the step is executed.
 #[derive(Deserialize, Serialize, Clone, Debug, Eq, PartialEq)]
 pub struct GethExecTrace {
+    // This one is returned directly by trace_block_by_number in rpc
+    /// True when the tx could not execute
+    pub invalid: Option<bool>,
```

Comment:
Why is this one an Option? (sorry if this was already the case when I last reviewed this code, I can't remember).

Reply:
It should not be an Option. I used Option in Rust in an incorrect way.